src/graphql/schema.graphql

```diff
@@ -159,12 +159,26 @@ type GamePieceMoveAction {
 
 type GamePieceRangedAttackAction {
   targetGamePiece: GamePiece!
+  resolved: Boolean!
+  resolvedAttacks: [ResolvedAttack!]
```

Comment:
This means once you have submitted the phase and all attacks have been resolved, you can query the API and get the list of `GamePieceActions` and see their results.

Note however we do not decrypt the dice rolls until the phase has been submitted and we actually resolve the actions. This is critical to stop the client from queueing up attacks and then reading their dice roll results before submitting the phase.